packages/suite-desktop/src-electron/libs/http-receiver.ts

```diff
@@ -56,7 +56,7 @@ export class HttpReceiver extends EventEmitter {
             {
                 pathname: '/oauth',
                 handler: this.oauthHandler,
-                origins: ['', '127.0.0.1'], // No referer is sent by Google and Dropbox
+                origins: ['', '127.0.0.1', 'https://www.dropbox.com'], // No referer is sent by Google, Dropbox sends referer when using Safari
```

Comment:
`https://` is extra here, we check if origin includes `new URL(referer).hostname`

```suggestion
                origins: ['', '127.0.0.1', 'www.dropbox.com'], // No referer is sent by Google, Dropbox sends referer when using Safari
```

Reply:
Done in e6cd6d8e0a0f99eb54f011d1e4fc40f376529c08.